File: src/main/java/fr/lewon/web/bot/entities/battle/TowerOfFameOpponentPremise.java
package fr.lewon.web.bot.entities.battle;

import com.fasterxml.jackson.annotation.JsonProperty;

public class TowerOfFameOpponentPremise {

	@JsonProperty("id_player")
	private String id;
	
	@JsonProperty("level")
	private int lvl;
	
	@JsonProperty("nb_challenges_played")
	private int nbChallengesPlayed;

	
	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public int getLvl() {
		return lvl;
	}

	public void setLvl(int lvl) {
		this.lvl = lvl;
	}

	public int getNbChallengesPlayed() {
		return nbChallengesPlayed;
	}

	public void setNbChallengesPlayed(int nbChallengesPlayed) {
		this.nbChallengesPlayed = nbChallengesPlayed;
	}

}

File: src/main/kotlin/fr/lewon/bot/hh/entities/battle/TowerOfFameOpponentPremise.kt
package fr.lewon.bot.hh.entities.battle

import com.fasterxml.jackson.annotation.JsonProperty

class TowerOfFameOpponentPremise {
    @JsonProperty("id_player")
    var id: String? = null
    @JsonProperty("level")
    var lvl = 0
    @JsonProperty("nb_challenges_played")
    var nbChallengesPlayed = 0

}